### DISCUSSION TITLE
make ble controller with esp32

### DISCUSSION THREAD

**Discussion by jnsw** [necroware/gameport-adapter#82] [Ideas]
is it possible to make use of this for creating a ble adapter für using nearly anywhere with ble? like on the phone or similar?

i have the sidewinder 3d pro and wnted to make use of it

**Comment by Rikki-Tikki-Tavi-12** [comment]
Old controllers draw a lot of power for mobile devices. You may get disappointing amounts of play time out of a battery pack. You can already use the project on any Android smartphone just by connecting it by usb. Though, again, it will draw a noticeable amount of power.
